src/software/ai/hl/stp/tactic/receiver/receiver_fsm_test.cpp

```diff
@@ -0,0 +1,119 @@
+#include <gtest/gtest.h>
+
+#include "shared/constants.h"
+#include "software/ai/hl/stp/action/move_action.h"
+#include "software/ai/hl/stp/tactic/receiver/receiver_tactic.h"
+#include "software/geom/algorithms/distance.h"
+#include "software/test_util/test_util.h"
+
+class OneTimeShotDirectionTest
```

Comment:
nit: can you rename this to OneTouchShotDirectionTest

Reply:
fixed